docs/source/api/container_index.rst

```diff
@@ -13,22 +13,23 @@ xexpression API is actually implemented in ``xstrided_container`` and ``xcontain
 
 .. toctree::
 
-   xcontainer
    xaccessible
-   xiterable
+   xadapt
    xarray
    xarray_adaptor
+   xbroadcast
    xchunked_array
-   xtensor
-   xtensor_adaptor
+   xcontainer
    xfixed
-   xoptional_assembly_base
+   xfunctor_view
+   xindex_view
+   xiterable
+   xmasked_view
    xoptional_assembly
    xoptional_assembly_adaptor
-   xmasked_view
-   xview
-   xstrided_view
-   xbroadcast
-   xindex_view
-   xfunctor_view
+   xoptional_assembly_base
    xrepeat
+   xstrided_view
+   xtensor
+   xtensor_adaptor
+   xview
```

Comment:
The order of these files should not be changed. The idea is that first come the base classes implementing the various concepts, and then the implementation classes; then the views.

Reply:
I first glance I did not get the idea. Reverted!